<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRefPropertiesWidget</class>
 <widget class="QWidget" name="XRefPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_type_cb"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_display_gb">
     <property name="title">
      <string>Représentation:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Positionner :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_snap_to_cb"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>XRef Vertical Offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_offset_sb">
          <property name="toolTip">
           <string>10px corresponds to 1 tile displacement</string>
          </property>
          <property name="whatsThis">
           <string>Set Vertical Offset for the Cross References. 10px corresponds to 1 tile displacement.</string>
          </property>
          <property name="specialValueText">
           <string>Default - Fit to XRef height</string>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="suffix">
           <string notr="true">px</string>
          </property>
          <property name="minimum">
           <number>40</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>XRef slave position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_xrefpos_cb"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="m_display_has_contacts_rb">
            <property name="text">
             <string>Affiche&amp;r en contacts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_display_has_cross_rb">
            <property name="text">
             <string>Afficher en croix</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Label des références croisées</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>48</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Maitre</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_master_le">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string>%f-%l%c</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>48</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Esclave</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_slave_le">
               <property name="text">
                <string>(%f-%l%c)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Créer votre propre texte en vous aidant des variables suivantes :
%f : le numéro de folio 
%F: le label de folio
%l : le numéro de ligne
%c : le numéro de colonne
%M: Installation 
%LM: Localisation </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_cross_properties_gb">
     <property name="title">
      <string>Option d'affichage en croix</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="m_show_power_cb">
        <property name="text">
         <string>Afficher les contacts de puissance dans la croix</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Préfixe des contacts de puissance :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Préfixe des contacts temporisés :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_delay_prefix_le"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_power_prefix_le"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Préfixe des contacts inverseurs :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="m_switch_prefix_le"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_type_cb</tabstop>
  <tabstop>m_snap_to_cb</tabstop>
  <tabstop>m_offset_sb</tabstop>
  <tabstop>m_xrefpos_cb</tabstop>
  <tabstop>m_display_has_contacts_rb</tabstop>
  <tabstop>m_display_has_cross_rb</tabstop>
  <tabstop>m_master_le</tabstop>
  <tabstop>m_slave_le</tabstop>
  <tabstop>m_show_power_cb</tabstop>
  <tabstop>m_power_prefix_le</tabstop>
  <tabstop>m_delay_prefix_le</tabstop>
  <tabstop>m_switch_prefix_le</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
